### ISSUE TITLE
Problem scrolling

### ISSUE THREAD

**Issue by IniterWorker** [callahanrts/slack-chat#39]
I can't scroll the message. The scroll bar is locked.

**Comment by Alexis-Martin** [comment]
Same problem

**Comment by brettneese** [comment]
Also experiencing this.

**Comment by mfrachet** [comment]
No way to manage this ?

**Comment by patriksima** [comment]
Hi,

I can't see whole last message, see screenshot. And I can't scroll either.

Ubuntu 16.04 LTS
Atom 1.10.2
SlackChat 3.4.2

![onpaste 20160909-092423](https://cloud.githubusercontent.com/assets/172640/18378801/4f6627d2-766f-11e6-8061-254798d801bc.jpg)

**Comment by bhaskardabhi-oviyum** [comment]
Facing same issue

**Comment by mikebz** [comment]
me too

**Comment by jaro-shell** [comment]
experiencing the same issue now

**Comment by Lienad** [comment]
I am also experiencing this just recently. I have tried force closing slack and even rebooting the computer

**Comment by hyliu1989** [comment]
facing the same problem. rebooting the mac or restarting the slack app does not help.

**Comment by jasonyost** [comment]
Same issue, suddenly cannot scroll to most recent messages in Slack app. Have tried restarting app and computer with same results. Mobile app shows the most recent messages.

**Comment by xlts** [comment]
Same with me, I would get rid of the problem by reloading the app on mac (command + r), but now it won't work

**Comment by tirkarthi** [comment]
Had the same issue. I sent myself a message and scrolled to have the scroll working across the app. But when I restarted the app the issue is still present. Reported this GitHub issue to slack so that they can see the issue with original slack client visually.

Edit : It worked fine first time but sending a message to myself again after restart doesn't fix the scroll now 😕  
Clarified the report.

**Comment by bhaskardabhi** [comment]
Dnt knw if repi is still maintained. 😕. Last commit 1 yr ago and one PR pending.

**Comment by revanaii** [comment]
same here on the latest macOS, the app is unusable at the moment

**Comment by aleksmatula** [comment]
I'm having the same issue

**Comment by tirkarthi** [comment]
Response from Slack as below : 

> We've had some other users report a similar issue when using our legacy app and, in these cases, an update to our latest app (v2.5.2) has resolved the issue. Would you mind giving this a try please? You will first need to uninstall the current app by quitting it and dragging it from Applications to the Trash. Once that's done, you can download the updated version from: https://slack.com/downloads/mac or from the app store.

Edit : Fixes the issue.

**Comment by sparr** [comment]
While upgrading does resolve the symptom, it seems implausible that so many people would start experiencing this issue on the same day after five months of no comments here unless something changed on the Slack servers. It would be nice if the devs would fix/undo/whatever that change/bug on their end.

**Comment by sparr** [comment]
NOTE: This repository is for an Atom plugin, not for the official Slack chat app/client. I, like many others above, mistakenly came to this issue while searching for reports of this issue with the official client.

**Comment by naolenikhil** [comment]
I am having the same problem. Did anyone find a solution/workaround?

**Comment by mikebz** [comment]
Yes I upgraded to the app from the App Store and it now works fine.

**Comment by sparr** [comment]
I recognize that I also made this mistake...

Please stop commenting here about a bug in the official Slack client. This repository is for a different piece of software.

**Comment by yuchiko** [comment]
I'm having the same issue

**Comment by ariswift** [comment]
Likewise, just noticed it in the last 30 minutes or so.  I'm using Slack through Chrome, not the app.

**Comment by mmenashe** [comment]
same here, it used to work well

**Comment by SerenaSq2** [comment]
still doesn't work...

**Comment by antwonjon** [comment]
Same issue

**Comment by antwonjon** [comment]
Completely restarting the application did work though

**Comment by IniterWorker** [comment]
But it's a temporary solution, @antwonjon. If I'm correct about it.

**Comment by imrimt** [comment]
Running into this issue with Slack app on Linux (Ubuntu 18.04) as well. Scrolling would stop working after using for a while. Temporary workaround is `Ctrl + R` (or whatever the refresh shortcut is); however, I would like to see a permanent fix at some point.

**Comment by maularamadhan** [comment]
Logout and login again solved the problem for me.

**Comment by easymoneymouss** [comment]
FIRST AND FOREMOST - Try logging out then back in. This should magically take care this issue, otherwise try tweeting Trump or JA Rule to see what if they can offer any help. :)

**Comment by abhinair247** [comment]
Logging in and out... Worked for me too..

**Comment by ersingenel** [comment]
Same here.

**Comment by brettcave** [comment]
I came here to report the slack scrolling issue but noticed a comment that this is not for the slack client. But it keeps getting +1'd here.... so...

my scroll isn't working either.

Suggest reporting to Slack directly, e.g. https://get.slack.help/hc/en-us (not sure if this is the best place).
